<launch>
	<arg name = "useVision" default = "true" />

	<include file="$(find awesomo)/launch/turtlebot_spawn.launch">
	</include>

	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/>
	<node pkg="robot_state_publisher" name="robot_state_publisher" type="robot_state_publisher" />

	<include file="$(find awesomo)/launch/ar_tag.launch" if="$(arg useVision)"/>
	<node name="Controller" pkg="awesomo" type="Controller" args="$(arg useVision)"/>
	<node pkg="awesomo" name="Planner" type="Planner" output="screen" args="$(arg useVision)"/>
	<node pkg="awesomo" name="threatGen" type="threatGen" output="screen" args="$(arg useVision)"/>
</launch>